### ISSUE TITLE
Should keywords be regular arguments with default value instead in rendering function?

### ISSUE THREAD

**Issue by MaximeBouton** [sisl/AutoViz.jl#6]
Some of the rendering function have keywords arguments, `render_vehicle` for example has `color_arrow`. When implementing a custom render function, we must use `add_instruction!(rm::RenderModel, f::Function, arr::Tuple; incameraframe::Bool=true )`.

The current implementation does not allow for passing keyword arguments for `f`. My suggestion is that we replace the keywords by regular arguments with default values. 

Another possibility is to change `add_instruction!` method to `add_instruction!(rm::RenderModel, f::Function, arr::Tuple, kwarr::Dict{Any, Any}; incameraframe::Bool=true)` but it might be complicated.

**Comment by tawheeler** [comment]
Yeah - we could change `add_instruction!` to `add_instruction!(rm::RenderModel, f::Function, arr::Tuple; incameraframe::Bool=true, kwargs...)` and then change all of the rendering functions.
